src/components/FileTree/index.tsx

```diff
@@ -134,10 +139,14 @@ export const buildFileTree = (
 
 export type PublicProps = {
   onSelect: FileTreeNodeProps['onSelect'];
+  versionId: number;
+};
+
+type PropsFromState = {
   version: Version;
```

Comment:
In regards to making this `Version | void`, I don't think it's important for this patch. It will make more work for https://github.com/mozilla/addons-code-manager/issues/442 but that's fine.